### PR TITLE
[WIP] Setup linting & fix lint errors

### PR DESCRIPTION
**Pull request by satyarohith**
Do not merge this yet. And please do suggest any code styles you would like to tweak.

- [x] `aws`
- [x] `do`
- [x] `dig`
- [x] `echo`
- [x] `route53`
- [x] `npm-example`
- [ ] `billing`
- [ ] `gcloud`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "command-sets",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "git@github.com:satyarohith/command-sets.git",
+  "author": "Satya Rohith <me@satyarohith.com>",
```
Comment (rabbah) [inline package.json]: We’re using Apache license.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "command-sets",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "git@github.com:satyarohith/command-sets.git",
```
Comment (rabbah) [inline package.json]: This needs to be fixed.

File: package.json (hunk 1 above)
Comment (satyarohith) [reply package.json]: Updated!

File: package.json (hunk 2 above)
Comment (satyarohith) [reply package.json]: Fixed.

File: route53/src/listhosts.js [hunk 3]
```diff
@@ -38,42 +41,45 @@ function compactResponse(data, type, match) {
 }
 
 async function _command(params, commandText, secrets = {}) {
-  const {
-    type = false,
```
Comment (satyarohith) [inline route53/src/listhosts.js:42]: @ericswildens Do we need these? I couldn't find where they were used so I removed them.

**Comment by satyarohith** [comment]
It isn't ready for review. Clicked it by mistake.

**Comment by satyarohith** [comment]
I will reopen once I get back to working on this.

**Pull request closed without merging**